README.md

```diff
@@ -918,6 +918,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Harsh Deep](https://github.com/HarshDeep61034)
   - [Harsh Dhariwal](https://github.com/harsh-dhariwal)
   - [Harsh Kapoor](https://github.com/Harsh-Kapoorr)
+  - [Haris Khan](https://github.com/harisdev-netizen)
```

Comment:
Please add your name in line 916, after Hariket Sukesh Kumar Sheth.
Let us know if you need any help doing that. 🙂

Reply:
Ok. Thanks for the heads up.